### PR TITLE
Adding prod squad norms

### PR DESCRIPTION
**Pull request by NickHeiner**
/cc @askldjd @mdbenjam @mogrenAtWork @amprokop @joofsh @laurjpeterson based on our conversation yesterday. Please let me know if this reflects your understanding.

### CODE REVIEW

File: docs/prod-squad.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+This procedure only needs to be done during business hours. When you start work in the morning, review errors that happened after close of business the prior day.
+
+### Backup
+If you are unavailable at any time during your prod squad rotation, whether it's because you're on vacation or out for an appointment in the afternoon, you're responsible for finding someone to replace you.
+
+## Who is on prod squad
+All engineers with VA email addresses participate in prod squad. The VA email address gives the ability to access our production systems.
+
+If you are on prod squad, you must have access to the following:
```
Comment (NickHeiner) [inline docs/prod-squad.md]: @askldjd am I missing anything on this list?

File: docs/prod-squad.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+1. PagerDuty
+1. CloudWatch logs
+1. Sentry
+
+If you don't have access to everything on this list, talk to your team lead about getting access.
+
+## Responsibility for Errors
+| App | Team | Tech Lead |
```
Comment (NickHeiner) [inline docs/prod-squad.md]: @laurjpeterson did I miss any apps?

File: docs/prod-squad.md (hunk 1 above)
Comment (askldjd) [inline docs/prod-squad.md]: I read it over 3x, and it looks solid. I have no comment. :+1: :+1: :+1:

**Comment by NickHeiner** [comment]
I added some notes from https://github.com/department-of-veterans-affairs/appeals-pm/blob/master/%5B%5BReview%5D%5D/how-we-work/Prod%20Squad.md to consolidate that doc into this one.

**Pull request merged**